coffee_core/src/lib.rs

```diff
@@ -14,6 +14,7 @@ pub enum CoffeeOperation {
     /// Setup(core lightning root path)
     Setup(String),
     Show(String),
+    Nurse(),
```

Comment:
```suggestion
    Nurse,
```

this is also valid rust right?

Reply:
Yes it is.